text/0000-data-formats.md

```diff
@@ -0,0 +1,151 @@
+# RFC: Data Formats for OpenTelemetry
+
+# Overview
+OpenTelemetry is an observability system, which generates observations, and then sends that data to remote systems for analysis. Describing how  this observability data is represented and transmitted is critical part of the OpenTelemetry specification.
+
+**Proposed Layers**
+*   Metadata and Semantic Conventions
+*   Logical Structure
+*   Encodings
+*   Exchange Protocols
+
+
+# Metadata and Semantic Conventions
```

Comment:
Does this need to mention "metadata"?

Reply:
Probably not, I can remove that reference.